v6/v6_interface.ml

```diff
@@ -28,9 +28,9 @@ exception Missing_connection_details
 exception License_checkout_error of string
 
 (* dbg_str -> requested edition -> current params ->
-   new edition * new xapi params * new additional params *)
```

Comment:
As mentioned in another review, this is fine for now but we should change this type to a record in the long run.  `string * (string * string) list * (string * string) list * (string * bool) list` is not a descriptive type and there is a good chance of putting strings into the wrong list.